### DISCUSSION TITLE
How to graduate alpha version to beta version?

### DISCUSSION THREAD

**Discussion by SevenOutman** [semantic-release/semantic-release#2290] [Release workflow] [answered]
Hi, what should I do if I have both `alpha` release channel (internal-only) and `beta` release channel (for public) and want to graduate an alpha release to beta channel?

**Comment by travi** [comment, marked as answer]
creating a `beta` branch from the `alpha` branch should be enough to accomplish this goal if you are using the default configuration

**Reply by SevenOutman** [reply]
I see. Thanks!
